### ISSUE TITLE
Feature request: Have the option to switch cameras

### ISSUE THREAD

**Issue by yanderemoe** [signalapp/Signal-iOS#2401]
- [yes] I have searched open and closed issues for duplicates

----------------------------------------

### Bug description
During video call, I can't find any option to switch between front/back cameras

### Steps to reproduce
- Start a video call

**Actual result:** No way to switch camera
**Expected result:** Have the button to switch camera

### Device info

**Device**: iPhone 6
**iOS version**: 10.3.3
**Signal version:** 2.15.1.0

**Comment by NinebitX** [comment]
I think this issue can be closed. The requested feature has now been implemented in version 2.25.

**Comment by silverdr** [comment]
I seem to have version later than 2.25 (2.44.0) and the supposedly implemented feature is still nowhere to be found…

**Comment by kaktusztea** [comment]
It is implemented. Top right corner, two circle arrows icon.

**Comment by silverdr** [comment]
@kaktusztea - please check the attachment below:

<img width="320" alt="91255" src="https://user-images.githubusercontent.com/4529686/68976973-6ae60f00-07f7-11ea-87b9-4a957520d1b8.png">

**Comment by silverdr** [comment]
It is surely not in the top right corner. Rather in an unexpected place, replacing out of a suddan another button in the middle. Only after processing a screenshot I found it, while of course in this case I can't find the button it replaces… This means another issue needs to be open. Said screenshot below:

<img width="320" alt="91255" src="https://user-images.githubusercontent.com/4529686/68977581-da103300-07f8-11ea-91da-5b64038796d4.png">

**Comment by kaktusztea** [comment]
Sorry, I thought you want to create and send an image.
As you found - the icon is at the middle.
